core/src/main/java/org/infinispan/commands/control/UnlockAllControlCommand.java

```diff
@@ -0,0 +1,91 @@
+package org.infinispan.commands.control;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.commands.remote.CacheRpcCommand;
+import org.infinispan.factories.ComponentRegistry;
+import org.infinispan.remoting.transport.Address;
+import org.infinispan.transaction.xa.GlobalTransaction;
+import org.infinispan.util.ByteString;
+import org.infinispan.util.concurrent.CompletableFutures;
+import org.infinispan.util.concurrent.locks.LockManager;
+
+/**
+ * Unlocks all keys associate to a {@link GlobalTransaction}.
```

Comment:
`TxCompletionNotificationCommand` requires a `RemoteTransaction` to unlock the keys (I guess it would be fine to use it)
`LockControlCommand(unlock=true)` the command doesn't do anything when unlock=true.

Reply:
done